lib/mix/lib/mix/tasks/test.ex

```diff
@@ -362,7 +362,15 @@ defmodule Mix.Tasks.Test do
     if File.exists?(file) do
       Code.require_file file
     else
-      Mix.raise "Cannot run tests because test helper file #{inspect file} does not exist"
```

Comment:
You're totally right! We could stick with the old one. I'll remove the test for this too.